.travis.yml

```diff
@@ -67,15 +70,15 @@ matrix:
 
     # OSX {
     - stage: Upload level 1
-      os: osx
-      env: TOOLCHAIN=osx-10-11-hid-sections-lto CONFIG=Release LEVEL=1
+      osx_image: xcode8.3
```

Comment:
As far as I understand it's default now. I think there is no need to set it explicitly.

Reply:
Until they change the default :)